emergency_alerts_utils/insensitive_dict.py

```diff
@@ -69,10 +69,10 @@ def __init__(
         *,
         index,
         error_fn,
-        recipient_column_headers,
-        placeholders,
+        # recipient_column_headers,
```

Comment:
Is there a reason we're commenting these rather than removing them?

Reply:
Testing - will fully remove once functionality is verified in dev